test/e2e/bpmn.navigation.test.ts

```diff
@@ -40,20 +51,20 @@ describe('diagram navigation', () => {
   const pageTester = new PageTester({ pageFileName: 'rendering-diagram', queryParams: [], expectedPageTitle: 'BPMN Visualization - Diagram Rendering' }, 'navigation');
 
   const fileName = 'simple.2.start.events.1.task';
-  let viewportCenterX: number;
-  let viewportCenterY: number;
+  let containerCenterX: number;
```

Comment:
👍🏿 for consistency, as we decided to remove the 'viewport' naming in favor or bpmn-container